generators/python-v2/sdk/src/wire-tests/WireTestSetupGenerator.ts

```diff
@@ -353,6 +354,67 @@ def pytest_unconfigure(config: pytest.Config) -> None:
         return orgName;
     }
 
+    /**
+     * Builds the environment setup for the conftest.py file.
+     * Returns an object with imports and the parameter to use in the client constructor.
+     *
+     * If the IR has environments defined (single or multiple base URLs), we need to
+     * create a custom environment instance that points all URLs to WireMock.
+     * If no environments are defined, we use base_url directly.
+     */
+    private buildEnvironmentSetup(): { imports: string; param: string } {
+        const environments = this.ir.environments;
+
+        if (environments == null) {
+            // No environments defined - use base_url directly
+            return {
+                imports: "",
+                param: 'base_url="http://localhost:8080"'
+            };
+        }
+
+        const envConfig = environments.environments;
+
+        // Handle single base URL environment
+        if (envConfig.type === "singleBaseUrl") {
+            const environmentClassName = this.getEnvironmentClassName();
+            const modulePath = this.getModulePath();
+            return {
+                imports: `from ${modulePath}.environment import ${environmentClassName}`,
+                param: `environment=${environmentClassName}(base="http://localhost:8080")`
+            };
+        }
+
+        // Handle multiple base URLs environment
+        if (envConfig.type === "multipleBaseUrls") {
+            const environmentClassName = this.getEnvironmentClassName();
+            const modulePath = this.getModulePath();
+
+            // Build kwargs for all base URLs pointing to WireMock
+            const baseUrlKwargs = envConfig.baseUrls
+                .map((baseUrl) => `${baseUrl.name.snakeCase.safeName}="http://localhost:8080"`)
+                .join(", ");
+
+            return {
+                imports: `from ${modulePath}.environment import ${environmentClassName}`,
+                param: `environment=${environmentClassName}(${baseUrlKwargs})`
+            };
+        }
+
+        // Fallback to base_url
+        return {
+            imports: "",
+            param: 'base_url="http://localhost:8080"'
+        };
```

Comment:
nit: this looks like unreachable code so could probably replace with `assertNever(envConfig)`